### DISCUSSION TITLE
Will mojo have native support for flask/ django?

### DISCUSSION THREAD

**Discussion by Daniel-online** [modular/modular#1651] [Q&A]
I want to know how much support is directed at web mojo or if it will be up to the community to create its own web-framework

**Comment by Brian-M-J** [comment]
In the [community live stream](https://www.youtube.com/live/eeLmtENdPhw?si=aVJEjKI-Pk-dQ1T1) at around 29:54, it was mentioned that it's up to the community to build non-ML libraries.
